### ISSUE TITLE
Audio output

### ISSUE THREAD

**Issue by MontMontgomery** [Proto-App/Proto-Android#131]
That's it, just another component.
I saw that Audio Input has been added, so i think Audio Output should be a great idea

**Comment by foresterr** [comment]
Definitely. There was another issue for mic and speaker, but it was closed after mic and audio input were added.

Audio out to wav file would be great, live audio out to speaker would be awesome.

**Comment by MontMontgomery** [comment]
Yeah, and a buzzer should be great

**Comment by kjcnett** [comment]
Yeah definitely could use an 8ohm speaker for audio circuits.

**Comment by electron271** [comment]
Would be very useful

**Comment by TheRektafire** [comment]
Yeah but the DAC is locked inside the misc pack so it would be behind a paywall presumably 🙃 but I guess it would be nice to have nonetheless...

**Comment by electron271** [comment]
We have an ADC, would it really hurt to give us a DAC?

**Comment by TheRektafire** [comment]
> We have an ADC, would it really hurt to give us a DAC?

if you look on the list of features you'll see there's a DAC but it's part of the "analog pack" and costs 3.99 :( presumably the audio output would be put in there too sadly

**Comment by electron271** [comment]
$4 may not be much, but I think a DAC might be a bit more useful than an ADC in some cases
